src/server/helpers/metabrainzNotifications.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable camelcase */
+import {cacheJSON, getCachedJSON} from '../../common/helpers/cache';
+import config from '../../common/helpers/config';
+
+import log from 'log';
+import request from 'superagent';
+
+
+const METABRAINZ_NOTIFICATIONS_URL = 'http://metabrainz.org/notification';
```

Comment:
This would also be good to pass as a config value, useful for test servers and for local testing